blockchain/blockchain.go

```diff
@@ -447,7 +461,7 @@ func (bc *BlockChain) restartStateMigration() {
 		}
 
 		root := block.Root()
-		logger.Warn("Restart state trie migration", "blockNumber", number, "root", root.String())
+		logger.Warn("State migration is restarted", "blockNumber", number, "root", root.String())
```

Comment:
```suggestion
		logger.Warn("Continue to migration state trie", "blockNumber", number, "root", root.String())
```

Reply:
How about this?
I intended use "State migration` for consisenty of relevant logs.
```suggestion
		logger.Warn("State migration is continued", "blockNumber", number, "root", root.String())
```

Reply:
I think `restart` and `continue` is similar. We can go both ways. :)

Reply:
Ok. Off we go~!